lib/foreman_inventory_upload/async/upload_report_job.rb

```diff
@@ -9,22 +9,32 @@ def self.output_label(label)
 
       def perform(filename, organization_id)
         label = UploadReportJob.output_label(organization_id)
+        @filename = filename
+        @organization = Organization.find(organization_id)
+
+        unless @organization.owner_details&.fetch('upstreamConsumer')&.fetch('idCert')
+          logger.info("Skipping organization #{@organization}, no candlepin certificate defined.")
+          progress_output_for(label) do |progress_output|
+            progress_output.write_line("Skipping organization #{@organization}, no candlepin certificate defined.")
+            progress_output.status = "Task aborted, exit 1"
+          end
+          return
+        end
```

Comment:
Can you move this section to be called after the `Setting[:content_disconnected]` section?
so it would return earlier if the system is disconnected